docs/codeql/codeql-language-guides/codeql-for-ruby.rst

```diff
@@ -10,6 +10,7 @@ Experiment and learn how to write effective and efficient queries for CodeQL dat
 
    basic-query-for-ruby-code
    codeql-library-for-ruby
+   abstract-syntax-tree-classes-for-working-with-ruby-programs
```

Comment:
Should there be a `-  :doc:` entry below? There are some for the other items.

Reply:
Yes please. 🤦🏻‍♀️ I missed that - great catch.

Reply:
no problem, i'll merge this PR and address this issue in a followup PR.